set-local-dev-dummy-env.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+# To build and run site-search properly you need those dummy environment variables below
```

Comment:
Just write:

"Execute this to set all required environment variables, in order to launch the `service`."

In README.md, place a reference to this file.